.github/workflows/format_check_pr.yml

```diff
@@ -41,33 +41,12 @@ jobs:
               - '**/*'
               - '!**/*.md'
 
-      - name: Run airbyte-ci format fix all
-        uses: ./.github/actions/run-dagger-pipeline
-        with:
-          context: "pull_request"
-          docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
-          docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
-          sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
-          github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
-          subcommand: "format fix all"
 
       # This is helpful in the case that we change a previously committed generated file to be ignored by git.
       - name: Remove any files that have been gitignored
-        if: always()
         run: git ls-files -i -c --exclude-from=.gitignore | xargs -r git rm --cached
```

Comment:
I think this step was only relevant when we were making changes to files and didn't want to commit them, and can be removed 🤔 

Any files that are currently gitignored should still be formatted - I think if we run into failures for those here, we should commit them back, and then they're properly formatted and won't be touched again.

Reply:
@alafanechere I'm sanity checking my assumptions here just to make sure. Was thinking about generated files this morning and how if you format them properly and check them in (in one PR), then regenerate (in another PR), they will be unformatted again, so the CI will see changes in the remote. 

I guess if our autoformatting check doesn't already have any problems with any gitignored files this shouldn't be an issue. But we might want to run this new manual action off of this branch and see what happens. 

If there's an issue, I think I'd prefer to add `.gitignore` to the list of files to ignore when formatting, rather than handling it in the action, since the command would still return a non-zero exit code when doing things in the order we're doing them now